src/test/java/uk/gov/companieshouse/pscfiling/api/controller/impl/PscIndividualFilingControllerImplIT.java

```diff
@@ -475,23 +469,46 @@ void createFilingWhenCeasedOnDateBlankThenResponse201() throws Exception {
             .andExpect(jsonPath("$.reference_psc_id", is(PSC_ID)))
             .andExpect(jsonPath("$.register_entry_date", is(REGISTER_ENTRY_DATE.toString())))
             .andExpect(jsonPath("$.updated_at", is(FIRST_INSTANT.toString())))
-                .andExpect(jsonPath("$.country_of_residence").doesNotExist());
+            .andExpect(jsonPath("$.country_of_residence").doesNotExist());
         verify(filingMapper).map(dto);
     }
 
+    @Test
+    void createFilingWhenPropertyUnrecognisedThenResponse400() throws Exception {
+        final var body = "{" + PSC07_FRAGMENT.replace("ceased_on", "Ceased_on") + "}";
+        final var expectedError =
```

Comment:
Message content is irrelevant here as not used. Please change to clarify it is ignored.